src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -3143,5 +3146,31 @@ void GraphOptimizer::MatchSdpaKvCache(Graph &graph) {
     }
 }
 
+void GraphOptimizer::HintForLLMWorkloads(Graph &graph) {
```

Comment:
1. We already have modelType abstraction to incapsulate some hint regarding model structure https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/config.h#L41. I would propose extend it with LLM type
2. I would propose to use dedicate function to initialize LLM type: https://github.com/openvinotoolkit/openvino/blob/e9d42dd60b961af2c2d92abde0a65ace6696542b/src/plugins/intel_cpu/src/plugin.cpp#L212 instead of this pass.
The condition should be smy like:
op::util::has_op_with_type<PagedAttention>(model) || (op::util::has_op_with_type<ScaledDotProductAttention> &&  model->get_variables().size() > 0).